src/layer/layer.rs

```diff
@@ -328,6 +351,22 @@ pub enum ObjectType {
     Value(String),
 }
 
+impl ObjectType {
+    pub fn node(self) -> String {
```

Comment:
Do `ObjectType::node()` and `ObjectType::value()` need to `panic` instead of returning `Option<String>`? What is the reasoning behind this choice?

Reply:
You're right, it may be better cause it makes it more generically applicable, but I'll always call it with unwrap() or expect() then. The purpose is for things like `rdf:type` triples where we always know the object is going to be a node, and not a value.

Reply:
Got it. I'm not sure which is better, so I'll leave it up to you.